sass/components/_post.scss

```diff
@@ -24,6 +24,11 @@
             color: #d04444 !important;
         }
     }
+    .post{
+        .post__link{
```

Comment:
A bit of nitpicking, but would you mind adding an extra space here to be consistent? One of these days, we really need to get better CSS linting like we have for the JS code
```suggestion
    .post {
        .post__link {
```

Reply:
@hmhealey done:)